packages/framework-provider-aws-infrastructure/src/infrastructure/stacks/scheduled-commands-stack.ts

```diff
@@ -0,0 +1,56 @@
+import { BoosterConfig, ScheduledCommandEnvelope, ScheduleInterface } from '@boostercloud/framework-types'
+import { Stack } from '@aws-cdk/core'
+import { Rule, Schedule, RuleTargetInput } from '@aws-cdk/aws-events'
+import { LambdaFunction } from '@aws-cdk/aws-events-targets'
+import { Code, Function, IEventSource } from '@aws-cdk/aws-lambda'
+import * as params from '../params'
+import { APIs } from '../params'
+export interface ScheduledCommandStackMembers {
+  scheduledLambda: Function
+}
+
+export class ScheduledCommandStack {
+  public constructor(
+    private readonly config: BoosterConfig,
+    private readonly stack: Stack,
+    private readonly apis: APIs
+  ) {}
+
+  public build(): ScheduledCommandStackMembers {
+    const scheduledLambda = this.buildLambda('scheduled-task', this.config.scheduledTaskHandler)
```

Comment:
I would add a check here to see if there are any scheduled commands defined by the user. If not, it is not needed to create this lambda. 

We do the same thing in the AuthStack: We don't create anything if there are no roles defined.